packages/@ember/-internals/views/lib/mixins/view_support.js

```diff
@@ -143,11 +142,11 @@ let mixin = {
   /**
    Returns the current DOM element for the view.
 
-   @property element
-   @type DOMElement
-   @public
-   */
-  element: descriptor({
```

Comment:
Followup: We should flatten this class into `Component` so we don't need to use this decorator